tests/test_client.py

```diff
@@ -0,0 +1,450 @@
+import os
+import sys
+import base64
+
+from dxltieclient import *
+from tests.test_base import BaseClientTest
+from tests.test_value_constants import *
+from tests.mock_tieserver import MockTieServer
+
+sys.path.append(
+    os.path.dirname(os.path.abspath(__file__)) + "/../.."
+)
+
+class JsonDumpCallback(DetectionCallback):
+
+    def on_detection(self, detection_dict, original_event):
+        pass
+
+
+class TestSubscribeUnsubscribe(BaseClientTest):
+
+    def test_subscribeunsubscribe(self):
+
+        file_reputation_change_topic = "/mcafee/event/tie/file/repchange/broadcast"
+        cert_reputation_change_topic = "/mcafee/event/tie/cert/repchange/broadcast"
+        file_detection_topic = "/mcafee/event/tie/file/detection"
+        file_first_instance_topic = "/mcafee/event/tie/file/firstinstance"
+        #TIE_EVENT_FILE_PREVALENCE_CHANGE_TOPIC = "/mcafee/event/tie/file/prevalence"
```

Comment:
Both of these stray comments are left in because the TIE Client technically registers the topics, it just doesn't have any functions that use them. For now they do not need to be tested, imo - but I wanted to leave these comments in as a reminder if the functionality is added later.